modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements/evidence_submissions_controller.rb

```diff
@@ -42,8 +36,8 @@ def set_submission_attributes
           raise Common::Exceptions::RecordNotFound, params[:nod_id] unless @appeal
```

Comment:
wouldn't params[:nod_id] would need to be changed here as well?

Reply:
Why?